odl/test/trafos/fourier_test.py

```diff
@@ -858,5 +859,98 @@ def fhat(x):
     assert np.allclose(ft_f_n, fhat(recip_n.coord_vectors[0]))
 
 
+# ---- Non-uniform Fourier Transform ---- #
+def test_non_uniform_fourier_trafo_2d_call():
+    im_size = 512
+    shape = [im_size] * 2
+    samples = np.array(
+        np.where(np.random.normal(size=shape) >= 0),
+        dtype=float,
+    ).T
```

Comment:
Because you want the first dimension to correspond to the number of samples and the second the number of dimensions in your data. With `np.where` you get 3 lists, each corresponding to one dimension of your data.